### ISSUE TITLE
Tnfsf11 & Tnfrsf11b levels

### ISSUE THREAD

**Issue by slaulederkind** [mgijax/mammalian-phenotype-ontology#3376]
>If this is a request for a new term, please provide us with the following information.

>1. Preferred term label (e.g., asplenia)

1.  abnormal Tnfsf11 level
2.  increased Tnfsf11 level
3.  decreased Tnfsf11 level
4.  abnormal Tnfrsf11b level
5.  increased Tnfrsf11b level
6.  decreased Tnfrsf11b level

>2. Synonyms (e.g., absent spleen)

1.  abnormal osteoprotegerin ligand level; abnormal OPGL level;  abnormal RANKL level;  abnormal receptor activator of nuclear factor kappa B ligand level; abnormal TNF superfamily member 11 level
4.  abnormal osteoprotegerin level; abnormal Opg level; abnormal TNF receptor superfamily member 11b level

Also similar synonyms for the increased/decreased terms.

>3. Textual definition (the definition should be understandable even for non-specialists. Include a PubMed ID or URL(s) to refer to any relevant article(s) that provides information about the suggested term).

1.  anomaly in the amount of TNF superfamily member 11
2.  elevated concentration of TNF superfamily member 11
3.  reduced concentration of TNF superfamily member 11
4.  anomaly in the amount of receptor activator of nuclear factor kappa B ligand
5.  elevated concentration of receptor activator of nuclear factor kappa B ligand
6.  reduced concentration of receptor activator of nuclear factor kappa B ligand

>4. Any additional notes to clarify the meaning or use of the term. Please provide a PubMed ID or URL(s) for the source of this information.

PMID:20643942

>5. Suggested parent term (Please look in the hierarchy of the MP using the [MP Browser](http://www.informatics.jax.org/vocab/mp_ontology) or other browsers such as [OLS](http://www.ebi.ac.uk/ols/ontologies/mp) and find the place in the MP where the new term best fits)

1.  abnormal protein level (MP:0008469)
2.  abnormal Tnfsf11 level
3.  abnormal Tnfsf11 level
4.  abnormal protein level (MP:0008469)
5.  abnormal Tnfrsf11b level
6.  abnormal Tnfrsf11b level

>6. If you would like a nanoattribution, please indicate your ORCID id or provide a label such as MGI:sbello (organization, first initial-last name)

ORCID:0000-0003-3645-6803
--
